scenario.py

```diff
@@ -36,7 +38,8 @@ def __init__(self, params, experiment_path, scenario_id=1, n_repeat=1):
         self.y_test = y_test
 
         # Performance of the model trained in a distributed way on all partners
-        self.federated_test_result = []
+        self.mpl_test_score = None
+        self.mpl_nb_epochs_done = int
```

Comment:
This is just a copy of `self.mpl.test_score` ? If this is the case, I would remove those two attributes.

Reply:
Yes, that seems better you are right!

Reply:
Done